### DISCUSSION TITLE
Welcome to fusin_vegeta Discussions!

### DISCUSSION THREAD

**Discussion by cylas017** [cylas017/fusin_vegeta#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by cylas017** [comment]
test alexandre

**Reply by cylas017** [reply]
@alexandreL012

**Comment by AlexandreL012** [comment]
@cylas017
